README.md

```diff
@@ -978,7 +978,7 @@ Usage: `nerdctl logout [SERVER]`
 ### :whale: nerdctl network create
 Create a network
 
-:information_source: To isolate CNI bridge, [CNI isolation plugin](https://github.com/AkihiroSuda/cni-isolation) needs to be installed.
+:information_source: To isolate CNI bridge, CNI plugin [firewall](https://www.cni.dev/plugins/current/meta/firewall/)(>= `v1.1.0`), or [CNI isolation plugin](https://github.com/AkihiroSuda/cni-isolation) needs to be installed.
```

Comment:
```suggestion
:information_source: To isolate CNI bridge, CNI plugins v1.1.0 or later needs to be installed.
```

Probably no need to mention the deprecated isolation plugin

Reply:
SGTM